### PR TITLE
Send event data about pod start failure

### PR DESCRIPTION
**Pull request by machacekondra**
This PR add possibility to collect information about failures during pod management actions. Currently we store this data in-memory so it won't persist node crash. So event data will be lost in case it was not sent and node crashes. We may introduce some file based queue if needed.

Currently by default start actions is executed every 15s and hb data are send ever 60s, so we try to start pod 4 times in minute. And we also send such event data to k8s. We may want to introduce increase of timeout in case pod fail to start. 15s, 30s, 60s, 120s, etc. same k8s does. But it may not be needed in case we use systemd for this.

### CODE REVIEW

**Comment by machacekondra** [comment]
@masayag @pkliczewski @jakub-dzon @eloycoto PTAL

File: internal/workload/manager.go [hunk 1]
```diff
@@ -238,6 +268,11 @@ func (w *WorkloadManager) ensureWorkloadsFromManifestsAreRunning() error {
 				// Workload is not running - start
 				err = w.workloads.Start(&podWithPath.pod)
 				if err != nil {
+					w.eventsQueue = append(w.eventsQueue, &models.EventInfo{
```
Comment (jakub-dzon) [inline internal/workload/manager.go:271]: Are you planning to add events related to pod `run` failures as well in a separate PR?

File: internal/workload/manager.go [hunk 2]
```diff
@@ -238,6 +268,11 @@ func (w *WorkloadManager) ensureWorkloadsFromManifestsAreRunning() error {
...
+						Message: err.Error(),
+						Reason:  "Started",
```
Comment (jakub-dzon) [inline internal/workload/manager.go:273]: Shouldn't it be `NotStarted`? IMO `Started` would describe event of successful start of a pod.

Comment (machacekondra) [reply internal/workload/manager.go:273]: Right. Actually k8s uses `Failed`, so I think I will reuse it.

File: internal/workload/manager.go (hunk 1 above)
Comment (machacekondra) [reply internal/workload/manager.go:271]: Sure I can.

**Review by pkliczewski — COMMENTED** [review]
/lgtm

File: internal/workload/manager.go (hunk 1 above)
Comment (pkliczewski) [reply internal/workload/manager.go:271]: We can do it in separate PR

**Review by jakub-dzon — APPROVED** [review]

**Pull request merged**